src/processes.c

```diff
@@ -2463,7 +2490,7 @@ static int ps_read (void)
 				procentry[i].pi_ru.ru_utime.tv_usec / 1000;
 
 			/* tv_usec is nanosec ??? */
-			pse.counters.cpu_system = procentry[i].pi_ru.ru_stime.tv_sec * 1000000 +
+			pse.total_processesers.cpu_system = procentry[i].pi_ru.ru_stime.tv_sec * 1000000 +
```

Comment:
Is `total_processesers` a typo? Or an over-eager `s/count/total_processes/`?

Reply:
Yeah a find and replace routine gone a little wrong. Reverted.